store/copr/store.go

```diff
@@ -125,6 +125,8 @@ func getEndPointType(t kv.StoreType) tikvrpc.EndpointType {
 		return tikvrpc.TiFlash
 	case kv.TiDB:
 		return tikvrpc.TiDB
+	case kv.TiFlashMPP:
```

Comment:
By setting this, client-go will use appropriate rpcCtx, and copTask will only be sent to tiflash_mpp nodes. Check [here](https://github.com/tikv/client-go/pull/481/files#diff-0c36d07fbd1fc71c850ff57d323fe12bdf7f2c053ef71b7fec5bb44358f1070dR865)